<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createProfileDialog</class>
 <widget class="QWidget" name="createProfileDialog">
  <property name="modal" stdset="0">
   <bool>false</bool>
  </property>
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>721</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Create New Profile</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>No Registraion Required!!!</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Just Enter A Online User Name and Log in!</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="rogueUserNameLabel">
     <property name="text">
      <string>User Name: ( 4 to 23 characters )</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="userNameEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="moodMessageLabel">
     <property name="text">
      <string>Mood Message: ( Up to 27 characters )</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="moodMessageEdit">
     <property name="text">
      <string>Let's Communicate!</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="loginButton">
     <property name="text">
      <string>Login!</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
